### ISSUE TITLE
c3.conf file problerm

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#141]
**Reported by muglerj on 23 Sep 2005 17:08 UTC**
```
When using an odd nodename like 'b01n035', the c3.conf
file comes out in a very strange order.
```

**Comment by dikim33** [comment]
**Comment by naughtont on 23 Sep 2005 18:50 UTC**
```

Actually, the ordering is whatever the %hash order is [from
get_machine_listing()].  The sort() routine in the
post_clients blows up from some reason with this naming
scheam.  I'm going to take a look at this.
```

**Comment by dikim33** [comment]
**Comment by naughtont on 1 Aug 2006 18:21 UTC**
I will not have time to look at this bug for the 5.0 timeframe.

**Comment by dikim33** [comment]
**Comment by bli on 4 Aug 2006 02:00 UTC**
Thomas, I have moved the milestone to 5.1, please have a look at this post 5.0.  Thanks!

**Comment by dikim33** [comment]
**Modified by jparpail on 11 Apr 2007 12:00 UTC**

**Comment by dikim33** [comment]
**Modified by valleegr on 30 Jun 2008 06:42 UTC**

**Comment by dikim33** [comment]
**Comment by valleegr on 5 Oct 2010 17:59 UTC**
(In [9307]) This check-in fixes #143 - Node are correctly sorted when generating the C3 
configuration file.
